packages/gestalt/src/Popover.js

```diff
@@ -95,7 +95,7 @@ export default function Popover({
   color = 'white',
   role = 'dialog',
   shouldFocus = true,
-  showCaret = false,
+  _deprecatedShowCaret,
```

Comment:
you removed the `= false`

Reply:
this is prob failing the snapshot. i'd bring it back

Reply:
DONE